airbyte-integrations/bases/base-normalization/normalization/transform_catalog/catalog_processor.py

```diff
@@ -67,6 +67,7 @@ def process(self, catalog_file: str, json_column_name: str, default_schema: str)
                 f"WARN: Resolving conflict: {conflict.schema}.{conflict.table_name_conflict} "
                 f"from '{'.'.join(conflict.json_path)}' into {conflict.table_name_resolved}"
             )
+        # TODO: open file to write DBML file
```

Comment:
@sspaeti / @marcosmarxm can try looking into producing the final .dbml file at the same location as the source.yaml file